src/pages/CardLayout/Subscribe.vue

```diff
@@ -0,0 +1,182 @@
+            <Input
+              class="apply__input"
+              name="email"
+              placeholder="e.g. hunter2@hackthe6ix.com"
+              label="Email"
+              :onChange="handler"
+              :value="email"
+            />
+            <Input 
+              type="checkbox"
+              class="apply__input"
+              name="acceptance"
+              label="I want to receive occasional emails about updates and news"
+              :onChange="handler"
+              :value="acceptance"
+            />
+        </div>
+        <div class="apply__page">
+          <h2 class="apply__subtitle">Thanks For Signing Up!</h2>
+        </div>
+      </div>
+    </div>
+    <div class='apply__controls'>
+      <Button class='apply__button' v-show='page == 0' :click='submit' :disabled='(!valid())'>Submit</Button>
+    </div>
+  </Card>
+</template>
+
+<script>
+  import { Card, Input, Button } from '@components';
+  import axios from 'axios';
+  const email_reg_exp = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+  export default {
+    name: 'Subscribe',
+    path: '/subscribe',
+    components: {
+      Input,
+      Button,
+      Card
+    },
+    data() {
+      return {
+        email: '',
+        source: this.$route.query.src,
+        acceptance: '',
+        confirmation: false,
+        height: 0,
+        page: 0,
+      };
+    },
+    mounted() {
+      window.addEventListener('resize', this.pageHeight, { passive: true });
+    },
+    beforeDestory() {
+      window.removeEventListener('resize', this.pageHeight, { passive: true });
+    },
+    updated() {
+      this.pageHeight();
+    },
+    methods: {
+      handler({ target }) {
+        if(target.type === 'checkbox') { this[target.name] = this.acceptance == "on" ? "off" : "on"; }
+        else {  this[target.name] = target.value; }
+      },
+      pageHeight() {
+        this.$nextTick(() => {
+          const page = document.querySelectorAll('.apply__page')[this.page];
+          this.height = page.clientHeight;
+        });
+      },
+      shiftPages() {
+        const pages = Array.from(document.querySelectorAll('.apply__page'));
+        pages.forEach((page, i) => {
+          const current = this.page === i;
+          page.style.transform = (
+            `translateX(${ this.page * -100 }%) translateX(${ this.page * -60 }px)`
+          );
+          page.style.opacity = current ? 1 : 0;
+        });
+        this.pageHeight();
+      },
+      next() {
+        this.page++;
+        this.shiftPages();
+      },
+      valid() {
+        return email_reg_exp.exec(this.email) !== null && this.acceptance === "on";
+      },
+      submit() {
+        if(!this.valid()) { return; }
+        const query = `
+        mutation {
+          subscribeMailingList(email: "${this.email}", source: "${this.source}") {
```

Comment:
NOOO, use variables next time xD (We can fix this when we make a re-usable fetch thing